### PR TITLE
Mage Tower Start to 3F

### PR DESCRIPTION
**Pull request by vtange**
Standardized Object names

If proofreading, don't bother scrutinizing every {memo X} line. I only edited a bunch of them to just make myself feel better/more coherent. 

**suggestion to IMCgames**

Having Grita follow me around is cute and all, but it's kind of weird when she still follows you during a boss fight. She totally gets hit by the boss and me when she should be hiding off in the corner or something.

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -1022,7 +1022,7 @@ ETC_20150317_001021	Swift-footed Weaver
 ETC_20150317_001022	Field Chapperition Skill Dummy
 ETC_20150317_001023	Abysmal Pond
 ETC_20150317_001024	Abysmal Rodelin
-ETC_20150317_001025	It has been said that the Helgacercle was made by the servants of the demon realm's aristocrats.
+ETC_20150317_001025	It has been said that the Helgasercle was made by the servants of the demon realm's aristocrats.
```
Comment (terrapin47) [inline ETC.tsv]: "the Helgasercle" -> "Helgasercle"

File: ETC.tsv [hunk 2]
```diff
@@ -13627,7 +13627,7 @@ ETC_20150729_013628	Defeat Abomination!
 ETC_20150729_013629	{memo X}Goddess Zemyna, the Goddess of Earth..
 ETC_20150729_013630	The delegates were sieged by the Werewolf{nl}Defeat the Werewolf!
 ETC_20150729_013631	Special Scent
-ETC_20150729_013632	The Jewel of Prominence is going crazy.
+ETC_20150729_013632	The Jewel of Prominence is unstable.
```
Comment (terrapin47) [inline ETC.tsv:13630]: Nice :+1:

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -4871,74 +4871,74 @@ QUEST_LV_0100_20150317_004870	Hand over Big Cockatrice tails to Tara Miles
...
 QUEST_LV_0100_20150317_004872	Obtain Big Cockatrice tails
 QUEST_LV_0100_20150317_004873	Someone is looking for you
-QUEST_LV_0100_20150317_004874	There is a person who is looking for you at Fedimian. Go to the gigantic statue of the Goddess at Fedimian.
+QUEST_LV_0100_20150317_004874	$There is a person who is looking for you at Fedimian. Go to the gigantic Goddess statue at Fedimian.
 QUEST_LV_0100_20150317_004875	{memo X}
 QUEST_LV_0100_20150317_004876	{memo X}
 QUEST_LV_0100_20150317_004877	{memo X}
 QUEST_LV_0100_20150317_004878	Go to the Mage Tower
-QUEST_LV_0100_20150317_004879	Grita told you at the Goddess of Gabija is blocking the attacks of the demons at the Mage Tower. Go to the Mage Tower to help the Goddess.
+QUEST_LV_0100_20150317_004879	$Grita told you at the Goddess Gabija is resisting the attacks of the demons at the Mage Tower. Go to the Mage Tower to help the Goddess.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "told you at" -> "told you that"

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -4871,74 +4871,74 @@ QUEST_LV_0100_20150317_004870	Hand over Big Cockatrice tails to Tara Miles
...
-QUEST_LV_0100_20150317_004914	Charge the jewel
-QUEST_LV_0100_20150317_004915	{memo X}If you unleash the Steam of Flame, it will bring the Essences of Flame. Fill the Essences to the 2nd pot.
-QUEST_LV_0100_20150317_004916	You filled the Essences into the Jewel of Prominence in full. Talk to Grita.
-QUEST_LV_0100_20150317_004917	You filled all the Essences. Ask Grita what to do next.
-QUEST_LV_0100_20150317_004918	Obtain the Sources of Flame by defeating monsters
-QUEST_LV_0100_20150317_004919	{memo X}The energy is not enough yet, so Grita told you that you should enhance the energy with the Sources of Flame. Defeat the monsters nearby and collect the Sources of Flame.
-QUEST_LV_0100_20150317_004920	You collected enough Sources of Flame. Talk with Grita again.
+QUEST_LV_0100_20150317_004884	$You successfully lit up the signal. Talk to Grita again.
+QUEST_LV_0100_20150317_004885	$Defeat the Rubblems
+QUEST_LV_0100_20150317_004886	$You lit up a signal to let the Goddess know that help is coming. Discuss with Grita on what to do next.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "Discuss what to do next with Grita."

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -4871,74 +4871,74 @@ QUEST_LV_0100_20150317_004870	Hand over Big Cockatrice tails to Tara Miles
...
+QUEST_LV_0100_20150317_004887	$Check the 1st Transport Magic Field
+QUEST_LV_0100_20150317_004888	$Grita told you that using the Transport Magic Field is the fastest way to go upstairs. Go to the 1st Transport Magic Field to see whether it is activating alright.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "activating alright" -> "working correctly" ?
or "see if it activates properly" ?

File: QUEST_LV_0100.tsv [hunk 6]
```diff
@@ -4871,74 +4871,74 @@ QUEST_LV_0100_20150317_004870	Hand over Big Cockatrice tails to Tara Miles
...
+QUEST_LV_0100_20150317_004890	$Defeat interfering Phyracons
+QUEST_LV_0100_20150317_004891	$Defeat interfering Drakes
+QUEST_LV_0100_20150317_004892	$Grita's face turned grim as she was checking the Transport Magic Field. Talk to Grita.
+QUEST_LV_0100_20150317_004893	$Activate the 2nd Transport Magic Field
+QUEST_LV_0100_20150317_004894	$Grita told you that the 1st Magic Field is badly broken and recommended to use the other, larger Transport Magic Field. Look for the 2nd Transport Magic Field and activate it.
+QUEST_LV_0100_20150317_004895	$You thought you could use the 2nd Transport Magic Field, but there was a trap set by the demons. Talk to Grita again.
+QUEST_LV_0100_20150317_004896	$Destroy the 2nd Transport Magic Field restraining Grita
+QUEST_LV_0100_20150317_004897	$Grita told you that with the danger present, the plan to directly reach Goddess Gabija via the Transport Magic Fields should be abandoned, and you should focus on the monsters rushing into the tower. Talk to Grita for more details.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "danger present" -> "present danger" or "current dangerous situation"

File: QUEST_LV_0100.tsv [hunk 7]
```diff
@@ -4871,74 +4871,74 @@ QUEST_LV_0100_20150317_004870	Hand over Big Cockatrice tails to Tara Miles
...
+QUEST_LV_0100_20150317_004902	{memo X}Activate the Barrier Device
+QUEST_LV_0100_20150317_004903	{memo X}The Barrier Device is at the end of the left hallway. Activate the Barrier Device.
+QUEST_LV_0100_20150317_004904	{memo X}You were able to activate the Barrier Device. Talk to Grita again.
+QUEST_LV_0100_20150317_004905	$You arrived on the 2nd floor of the Mage Tower. Talk to Grita about what to do next.
+QUEST_LV_0100_20150317_004906	$Search for the Jewel of Prominence
+QUEST_LV_0100_20150317_004907	$Find the Jewel of Prominence. Use the detector to look for the Jewel of Prominence on the 2nd floor of the Mage Tower.
+QUEST_LV_0100_20150317_004908	$You found the Jewel of Prominence. Talk to Grita.
+QUEST_LV_0100_20150317_004909	$You got the Jewel of Prominence. Talk with Grita about how you can input the power into the Jewel of Prominence.
+QUEST_LV_0100_20150317_004910	$Collect Flame Vapor
+QUEST_LV_0100_20150317_004911	$Grita told you that in order to complete the Jewel of Prominence, you should fill Flame Vapor into it. Fill it with the Flame Vapor that leaks in the Large Reading Room.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "fill Flame Vapor into it" -> "fill it with Flame Vapor"

File: QUEST_LV_0100.tsv [hunk 8]
```diff
@@ -4871,74 +4871,74 @@ QUEST_LV_0100_20150317_004870	Hand over Big Cockatrice tails to Tara Miles
...
+QUEST_LV_0100_20150317_004920	$You collected enough Flame Sources. Talk to Grita again.
 QUEST_LV_0100_20150317_004921	{memo X}Obtain %s from the monsters nearby
-QUEST_LV_0100_20150317_004922	Obtain the Sources of Flame
-QUEST_LV_0100_20150317_004923	Now, one more step to go before the Jewel of Prominence gets completed. Talk to Grita.
-QUEST_LV_0100_20150317_004924	Combine materials with the Jewel of Prominence in the flame generator
-QUEST_LV_0100_20150317_004925	Grita told you that since the Jewel of Prominence is not complete yet, you should combine materials with it in the flame generator. Combine the materials at the flame generator.
-QUEST_LV_0100_20150317_004926	You successfully completed the Jewel of Prominence. Talk to Grita.
-QUEST_LV_0100_20150317_004927	Grita is waiting at the third floor of Mage Tower. Talk to Grita.
+QUEST_LV_0100_20150317_004922	$Obtain Flame Sources
+QUEST_LV_0100_20150317_004923	$Now, one more step to complete the Jewel of Prominence. Talk to Grita.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "Now, there is"

File: QUEST_LV_0100.tsv [hunk 9]
```diff
@@ -4871,74 +4871,74 @@ QUEST_LV_0100_20150317_004870	Hand over Big Cockatrice tails to Tara Miles
...
-QUEST_LV_0100_20150317_004934	As you destroyed the valve, it exploded and the monsters disappeared, but it was really dangerous. Talk to Grita again.
-QUEST_LV_0100_20150317_004935	Destroy the valve of magic
-QUEST_LV_0100_20150317_004936	The power of the valve of magic was way more than you expected. You better discuss it with Grita.
-QUEST_LV_0100_20150317_004937	Look for the info on the valve of magic
-QUEST_LV_0100_20150317_004938	Grita told you that since Antares researched on the flow of the magic, the research information must be left somewhere. Research the books of Antares and find the information on the magic device.
-QUEST_LV_0100_20150317_004939	It seems that you found information on the flow of the magic. Talk to Grita.
+QUEST_LV_0100_20150317_004932	$Destroy the first Magic Control Valve
+QUEST_LV_0100_20150317_004933	$Grita told you to break the Magic Control Valve to stop Antares. Destroy the valve at the Laboratory.
+QUEST_LV_0100_20150317_004934	$As you destroyed the valve, it exploded and the monsters disappeared, it was really dangerous. Talk to Grita again.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: Did the monsters disappear or appear?
"As you destroyed the valve, it exploded and monsters appeared, which was really dangerous."

File: QUEST_LV_0100.tsv [hunk 10]
```diff
@@ -4988,13 +4988,13 @@ QUEST_LV_0100_20150317_004987	Use the Jewel of Prominence
...
 QUEST_LV_0100_20150317_004990	You received the revelation from the Goddess. Ask the Goddess what to do next.
-QUEST_LV_0100_20150317_004991	Now, you should destroy the 2nd valve. Talk to Grita.
-QUEST_LV_0100_20150317_004992	Grita told you that there is the second valve in the central control room. Go to the central control room.
-QUEST_LV_0100_20150317_004993	Antares touched the wrong part of the valve so the valve exploded. Talk to Grita.
-QUEST_LV_0100_20150317_004994	Check the activation device for the protective shell
-QUEST_LV_0100_20150317_004995	The activation device for the protective shell is an important equipment to protect the Mage Tower. Look carefully on it.
-QUEST_LV_0100_20150317_004996	Defeat Salamander
-QUEST_LV_0100_20150317_004997	Salamander suddenly disappeared as you tried to look at the activation device.
+QUEST_LV_0100_20150317_004991	$Now it's time to destroy the 2nd Valve. Talk to Grita.
+QUEST_LV_0100_20150317_004992	$Grita told you that the second valve in the Central Control Room. Go to the Central Control Room.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "second valve in the" -> "2nd Valve is in the" ?

File: QUEST_LV_0100.tsv [hunk 11]
```diff
@@ -4988,13 +4988,13 @@ QUEST_LV_0100_20150317_004987	Use the Jewel of Prominence
...
+QUEST_LV_0100_20150317_004993	$Antares touched the wrong part of the valve, so the valve exploded. Talk to Grita.
+QUEST_LV_0100_20150317_004994	$Check the Barrier Activation Device
+QUEST_LV_0100_20150317_004995	$The Barrier Activation Device is an important piece of equipment that protects the Mage Tower. Carefully inspect it.
+QUEST_LV_0100_20150317_004996	$Defeat Salamander
+QUEST_LV_0100_20150317_004997	$Salamander suddenly disappeared as you tried to inspect the Activation Device.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "The Salamander suddenly appeared" ?

File: QUEST_LV_0100.tsv [hunk 12]
```diff
@@ -5501,42 +5501,42 @@ QUEST_LV_0100_20150317_005500	Talk to the Wizard Master
...
-QUEST_LV_0100_20150317_005532	Talk to the Sealing Stone
-QUEST_LV_0100_20150317_005533	There's a strange sound coming from the Grand Library. Talk to the Seal Stone.
-QUEST_LV_0100_20150317_005534	Defeat the Belegg and Blindlem
-QUEST_LV_0100_20150317_005535	Seal stone says weird things like its soul is trapped and etc. It's hard to believe but defeat the Blindlem and Beleggs like the book wants.
-QUEST_LV_0100_20150317_005536	Talk to the Sealed Stone
-QUEST_LV_0100_20150317_005537	Defeated the Blindlem and Beleggs. Return to the Seal stone and release the trapped soul.
-QUEST_LV_0100_20150317_005538	Defeat the Blindlem
-QUEST_LV_0100_20150317_005539	Defeat the Belegg
+QUEST_LV_0100_20150317_005527	$Ask Cordelier about the unpleasant vibration that you felt a while ago
+QUEST_LV_0100_20150317_005528	$Check the bookshelf of Reading Room
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "in the Reading Room"

File: QUEST_LV_0100.tsv [hunk 13]
```diff
@@ -5501,42 +5501,42 @@ QUEST_LV_0100_20150317_005500	Talk to the Wizard Master
...
+QUEST_LV_0100_20150317_005529	$Check the bookshelf in Reading Room to find out about the vibration felt in the Reading Room.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "in the Reading Room"

File: QUEST_LV_0100.tsv [hunk 14]
```diff
@@ -6672,8 +6672,8 @@ QUEST_LV_0100_20150414_006671	{memo X}Luring/BURY/3/TRACK_BEFORE/SSN_HATE_AROUND
 QUEST_LV_0100_20150414_006672	{memo X}Explain about the Rexipher and the clues
 QUEST_LV_0100_20150414_006673	About the Altar here
 QUEST_LV_0100_20150414_006674	Activate Cetek Altar
-QUEST_LV_0100_20150414_006675	Grita wants to bring defese magic in preparation for the explosion when destroying the valve. Bring the core of Red Infro Rockor that is used for the defese magic.
-QUEST_LV_0100_20150414_006676	Collected and gace the core of Red Infro Rockor. Talk to Grita again.
+QUEST_LV_0100_20150414_006675	$Grita wants to prepare defensive magic to prepare against explosions from destroying the valves. Bring the cores of Red Infrorocktor that is used for the defense magic.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "against explosions that will come from destroying the valves"

"the cores of Red Infrorocktors, that are used for this defensive magic."

File: QUEST_LV_0100.tsv [hunk 15]
```diff
@@ -6712,23 +6712,23 @@ QUEST_LV_0100_20150428_006711	That's why I am thinking about setting fire.{nl}I
...
+QUEST_LV_0100_20150428_006726	$Please set this gem and defeat the monsters around it.{nl}We can collect their energy to use it to recharge the Barrier Device.
 QUEST_LV_0100_20150428_006727	{memo X}A suitable end for a crazy magician.{nl}It could have been dangerous without defensive magics.{nl}We may have touched it without knowing the first valve was destroyed..{nl}
 QUEST_LV_0100_20150428_006728	{memo X}Anyways, that's good.{nl}The tower is now safe, so let's go up.
-QUEST_LV_0100_20150428_006729	Ah, that's right. Let's also prepare defensive magics in case something happens.{nl}There was a magic written on Antare's materials that we can use when there are explosions.
-QUEST_LV_0100_20150428_006730	{memo X}I think I can find a way if I have the core of Red Infro Rocktor. {nl}I can't use magic but that does't mean I lost my knowledge.
-QUEST_LV_0100_20150428_006731	Good. We can use it one more time.{nl}
+QUEST_LV_0100_20150428_006729	$Ah, that's right. Let's also prepare some defensive magics in case something happens.{nl}There was a spell written in Antare's materials that we can use when there are explosions.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "use when there are explosions" -> "use to protect against explosions" ?

File: QUEST_LV_0100.tsv [hunk 16]
```diff
@@ -6738,51 +6738,51 @@ QUEST_LV_0100_20150428_006737	{memo X}And me.. I feel dizzy..
...
-QUEST_LV_0100_20150428_006741	{memo X}You gathered everything.{nl}Give them to me. Including the Jewel.{nl}I know the way to stabilize it fully so let's go to lab room.
-QUEST_LV_0100_20150428_006742	Good! We can activate the device now.
+QUEST_LV_0100_20150428_006741	{memo X}You gathered everything.{nl}Give them to me. Including the Jewel.{nl}I know the way to stabilize it fully so let's go to Laboratory.
+QUEST_LV_0100_20150428_006742	$Good! Now we can activate the device.
 QUEST_LV_0100_20150428_006743	{memo X}You..are the revelator who helped the Paladin Master right?{nl}My name is Grita. I came from the Mage Tower.
 QUEST_LV_0100_20150428_006744	{memo X}Help me. Gabija is in danger.{nl}We are gonna lose the tower to the demons if we don't receive help.
-QUEST_LV_0100_20150428_006745	There is a way to stop it but only you can do it. {nl}Please, come to Karsta battlefield in Fedimian outskirt.
+QUEST_LV_0100_20150428_006745	$Only you can stop all that from happening. {nl}Please, come to Karsta Hall Site at the Fedimian Suburbs.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "the Karsta Hall Site"

File: QUEST_LV_0100.tsv [hunk 17]
```diff
@@ -6738,51 +6738,51 @@ QUEST_LV_0100_20150428_006737	{memo X}And me.. I feel dizzy..
...
+QUEST_LV_0100_20150428_006778	$Don't worry. It won't hurt you or explode.
+QUEST_LV_0100_20150428_006779	$Phew.. {nl}We've almost completed the Jewel of Prominence!
 QUEST_LV_0100_20150428_006780	{memo X}I will collect the sources of the flame into the last pot.{nl}You can get them from the monster that has the power of fire.
-QUEST_LV_0100_20150428_006781	Gabija will recover her energy if we could make the Jewel of Prominence completely.
-QUEST_LV_0100_20150428_006782	Good!{nl}Now we just have to combine the flame into the jewel.
-QUEST_LV_0100_20150428_006783	Let's go to the fusion machine.{nl}We just have to put all the materials in it and the jewel will be completed when we activate the device.
-QUEST_LV_0100_20150428_006784	So this is the Jewel of Prominence. I can feel it's great energy.{nl}Let's hurry and give this to Gabija. There's a stairway up on the right side.
-QUEST_LV_0100_20150428_006785	To find the Jewel of Prominence, just use this detector.{nl}When you use this detector near the place where the jewel is located, the jewel will come out.
+QUEST_LV_0100_20150428_006781	$Gabija will recover her energy if we could complete the Jewel of Prominence.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv]: "could" -> "can"

File: QUEST_LV_0100.tsv [hunk 18]
```diff
@@ -6738,51 +6738,51 @@ QUEST_LV_0100_20150428_006737	{memo X}And me.. I feel dizzy..
 QUEST_LV_0100_20150428_006738	If it stays like this, the Jewel of Prominence may break.{nl}There must be a stabilizing device somewhere here. Let's go.
 QUEST_LV_0100_20150428_006739	{memo X}Ah, the stabilization has come to an end.{nl}I am okay. It's nothing. Just old memory coming back.
 QUEST_LV_0100_20150428_006740	The Jewel of Prominence has stopped going crazy.{nl}But, in order to stabilize it fully, we need the Essence of Flame from the fireplace in the middle of the tower.
```
Comment (terrapin47) [inline QUEST_LV_0100.tsv:6744]: Should change "going crazy" here too?

File: QUEST_LV_0100.tsv (hunk 9 above)
Comment (vtange) [inline QUEST_LV_0100.tsv]: korean text says it exploded and cleared away the monsters. so disappeared is correct

File: QUEST_LV_0100.tsv (hunk 18 above)
Comment (vtange) [inline QUEST_LV_0100.tsv:6744]: this line is changed in #1150

**Comment by terrapin47** [comment]
Great work, good to merge :+1:

File: QUEST_LV_0100.tsv [hunk 19]
```diff
@@ -4871,74 +4871,74 @@ QUEST_LV_0100_20150317_004870	Hand over Big Cockatrice tails to Tara Miles
...
+QUEST_LV_0100_20150317_004907	$Find the Jewel of Prominence. Use the detector to look for the Jewel of Prominence on the 2nd floor of the Mage Tower.
+QUEST_LV_0100_20150317_004908	$You found the Jewel of Prominence. Talk to Grita.
+QUEST_LV_0100_20150317_004909	$You got the Jewel of Prominence. Talk with Grita about how you can input the power into the Jewel of Prominence.
+QUEST_LV_0100_20150317_004910	$Collect Flame Vapor
+QUEST_LV_0100_20150317_004911	$Grita told you that in order to complete the Jewel of Prominence, you should fill it with Flame Vapor. Fill it with the Flame Vapor that leaks in the Large Reading Room.
+QUEST_LV_0100_20150317_004912	$You filled the Jewel of Prominence with Flame Vapor. Return to Grita.
+QUEST_LV_0100_20150317_004913	$You collected all the Flame Vapor. Ask Grita what to do next to complete the Jewel of Prominence.
+QUEST_LV_0100_20150317_004914	$Charge the Jewel
+QUEST_LV_0100_20150317_004915	{memo X}If you unleash the Flame Vapor, it will bring the Essences of Flame. Fill the Essences to the 2nd pot.
+QUEST_LV_0100_20150317_004916	$You filled the Jewel of Prominence with Essence of Fire. Talk to Grita.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: with Essence of Fire ->  with the Essence of Fire?

**Comment by imcgames** [comment]
:+1:

**Pull request merged**